src/styles/styles.css

```diff
@@ -0,0 +1,178 @@
+html {
+  font-family: Roboto, Arial, 'Helvetica Neue', 'Segoe UI', sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+}
+
+.card__content {
+  width: 198px;
+  height: 406px;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 160px;
+  height: 134px;
+  background-image: url(../images/imac.jpeg);
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  margin-left: 19px;
+  margin-top: 32px;
+}
+
+.card__title {
+  margin-top: 40px;
+  margin-left: 16px;
+  height: 36px;
+  width: 166px;
+}
+
+.card__title-text {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  margin-top: 4px;
+  margin-left: 16px;
+  line-height: 14px;
+}
+
+.card__code-text {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__rating {
+  display: flex;
+  height: 16px;
+  width: 166px;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 17px;
+  margin-top: 16px;
+}
+
+.card__rating-stars:nth-child(1) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-stars:nth-child(2) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-stars:nth-child(3) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-stars:nth-child(4) {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-stars:nth-child(5) {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__rating-text {
+  font-size: 10px;
+  line-height: 14px;
+  text-align: right;
+  margin-left: auto;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 16px;
+  margin-top: 24px;
+  margin-left: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+  margin-left: auto;
+  color: #060b35;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  margin-top: 16px;
+  margin-left: 16px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+}
```

Comment:
The `.card__buy` button has fixed dimensions which can lead to content overflow if the text is too long. Use padding instead of fixed height for better text accommodation and consider min-width for better responsiveness.